src/test/java/com/googleinterns/zoomtube/servlets/IconFeedbackServletTest.java

```diff
@@ -112,4 +129,61 @@ public void doPost_validRequest_shouldStoreInDatabase() throws IOException, Serv
     assertThat(iconFeedback.timestampMs()).isEqualTo(456);
     assertThat(iconFeedback.type().toString()).isEqualTo("GOOD");
   }
+
+  @Test
+  public void doGet_missingLectureId_shouldRespondWithBadRequest()
+      throws IOException, ServletException {
+    servlet.doGet(request, response);
+
+    verify(response).sendError(
+        HttpServletResponse.SC_BAD_REQUEST, /* message= */ "Missing lecture id parameter.");
+  }
+
+  @Test
+  public void doGet_noFeedback_shouldReturnNoIconFeedback() throws IOException, ServletException {
+    when(request.getParameter(IconFeedbackServlet.PARAM_LECTURE_ID)).thenReturn("123");
+
+    servlet.doGet(request, response);
+
+    verify(response).setContentType("application/json");
+    List<IconFeedback> result = getIconFeedbackFromJson(content.toString());
+    assertThat(result.size()).isEqualTo(0);
+  }
+
+  @Test
+  public void doGet_oneIconFeedback_shouldReturnOneIconFeedback()
+      throws IOException, ServletException {
+    Key lectureKey = KeyFactory.createKey(LectureUtil.KIND, 123);
```

Comment:
can you add an inline comment for the literal?

Reply:
+1, this also occurs in other lines too.

Reply:
fixed